apps/frontend/src/pages/projects/styles.module.css

```diff
@@ -1,7 +1,13 @@
-.label {
+.title {
 	margin: 0;
 }
 
+.projects-header {
+	display: flex;
+	justify-content: space-between;
+	min-height: 44px;
```

Comment:
Do we need this? can't it just use it's children size?
```suggestion
```

Reply:
I put it cause in figma height is exact 44px, but I have a bit less than 44px. Now I noticed, that the issue is with button component. It's height 42px, but according to Figma it should has `min-height` 44px.
So, sure, I'll rid of this `min-height` and add this style to button.